JSON_for_IO-Link.yaml

```diff
@@ -2191,16 +2191,16 @@ paths:
         "501":
           $ref: "#/components/responses/HTTP_501"
 
-  "/devices/{deviceAlias}/parameters/{index}/value":
+  "/devices/{deviceAlias}/parameters/{parameterIdent}/value":
```

Comment:
Why this terminology change? "index" comes from the IO-Link-Spec, "parameterIdent" not!

Reply:
You can find the reason why it got changed in the issue description (https://github.com/iolinkcommunity/JSON_for_IO-Link/issues/121). 
But it did not vanish completely, just got put into an overlaying schema in order to fulfill the open api specification.
```
parameterIdent:
      oneOf:
        - $ref: "#/components/schemas/parameterName"
        - $ref: "#/components/schemas/index"
```
Of course, I am open for any better namings.

Reply:
@roesekoSICKAG: Maybe I am missing something, but when I read  #121, I cannot see a reason why the name "index" must be changed.

Reply:
It's all about a limitation in the OpenAPI Specification: 
Both paths
```
"/devices/{deviceAlias}/parameters/{index}/value"
"/devices/{deviceAlias}/parameters/{parameterName}/value"
```
are considered identical and therefore invalid (see [this page](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#path-templating-matching) for a detailed explanation). 
To make things valid again, `paramterIdent` is used as kind of a placeholder, which represents one of `parameterName` or `index`.